op-tests/test-bls-ops.txt

```diff
@@ -0,0 +1,209 @@
+coinid 0x1234500000000000000000000000000000000000000000000000000000000000 0x6789abcdef000000000000000000000000000000000000000000000000000000 0x00001234 => FAIL
+
+; bls g1 subtraction
+; expects (G1point G1point ...)
+bls_g1_subtract 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb 0xa572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62ae28f75bb8f1c7c42c39a8c5529bf0f4e => 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb | 2857918
+; identity
+bls_g1_subtract => 0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 132812
+; G1 point has invalid length
+bls_g1_subtract 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6 => FAIL
+bls_g1_subtract 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb00 => FAIL
+bls_g1_subtract 0 => FAIL
+bls_g1_subtract ( 1 2 3 ) => FAIL
+
+; bls g1 multiplication
+; expects (G1point scalar)
+bls_g1_multiply 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb 2 => 0xa572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62ae28f75bb8f1c7c42c39a8c5529bf0f4e | 2154839
+; missing scalar argument
+bls_g1_multiply 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb => FAIL
+; G1 point has invalid length
+bls_g1_multiply 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6 => FAIL
+bls_g1_multiply 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb00 => FAIL
+; missing/invalid arguments
+bls_g1_multiply => FAIL
+bls_g1_multiply 0 => FAIL
+bls_g1_multiply ( 1 2 3 ) => FAIL
+; too many arguments
+bls_g1_multiply 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb 2 3 => FAIL
+
+; bls g1 negate
+; expects (G1Point)
+bls_g1_negate 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb => 0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb | 471259
+; G1 point has invalid length
+bls_g1_negate 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6 => FAIL
+bls_g1_negate 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bbdd => FAIL
+bls_g1_negate 0 => FAIL
+; invalid number of arguments
+bls_g1_negate 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb => FAIL
+bls_g1_negate => FAIL
+bls_g1_negate ( 1 2 3 ) => FAIL
+
+; bls g2 add
```

Comment:
is this different from `point_add`?

Reply:
point_add is g1 points

Reply:
Should all the g1 point operations be called `point_whatever` then?

Reply:
the direction this patch takes is to, instead, create an alias for `point_add` called `bls_g1_add`